src/components/data_display/index.ts

```diff
@@ -1,3 +1,8 @@
-import { routeDataViews, routeDataViewTable, routeDataViewDetail } from './routes'
+import {
+  routeDataViews,
+  routeDataViewTable,
+  routeDataViewDetail,
+  routeDataViewFilterList,
```

Comment:
"filterList" endpoint is for fetching the available options for Filters. It's not complete, so please ignore in this PR.